public/whiteboard/js/main.js

```diff
@@ -1,14 +1,103 @@
 /* global */
+let board
+let img
+let brush
+let cnv
+let bSize
+let colorPicker
+let currColor
+let sizeSlider
+let saveButton
+
+/* Socket info to send;
+    mouse position
+    current canvas (as png? encoded 64bit?)
+     */
+
+function createBrush(hex){
+  // loadPixels()
+  // img.loadPixels()
+  // for (let i=0; i< img.width; i++){
+  //   for (let j=0; j< img.height; j++){
+  //     // let loc = i+j*width
+  //     // img.pixels[loc] = color(hex)
+  //     setup(i, j, hex)
+  //     print("i: " + i + " j: " + j)
+  //   }
+  // }
+  // print(img.width)
+  // updatePixels()
+  // return img
+  tint(hex)
+  return img
+}
+
+function preload(){
+  img = loadImage('https://img.icons8.com/emoji/452/-emoji-sparkles.png')
```

Comment:
now that `brush` is going to be an instance of the `Brush` class, we can do something like this here
```js
img = loadImage('path/to/image.png')
brush = new Brush(img)
```